### PR TITLE
Scy/20.21.7/bump consistent eval and profile card

### PR DESCRIPTION
**Pull request by Scy-D2L**
As discussed in SLACK [#web-platform](https://d2l.slack.com/archives/C0PHG3QB0/p1624372327327000)

### CODE REVIEW

**Review by vix633 — APPROVED** [review]

File: package-lock.json [hunk 1]
```diff
@@ -1326,9 +1326,9 @@
       }
     },
     "@brightspace-ui-labs/grade-result": {
-      "version": "1.2.4",
-      "resolved": "https://registry.npmjs.org/@brightspace-ui-labs/grade-result/-/grade-result-1.2.4.tgz",
-      "integrity": "sha512-psU98f6qll4IliiiPmfxiu1xofFDWTQ7jMx7kCZx3DoKlmJ9Ne0sqoOdS8VvJIe/fI1pNbtS201/My/1nt32ig==",
+      "version": "1.2.5",
+      "resolved": "https://registry.npmjs.org/@brightspace-ui-labs/grade-result/-/grade-result-1.2.5.tgz",
+      "integrity": "sha512-vloc7RE6P2zH/BPl+QxvgtLec+FnVljkHSIW2PH/huSk89xj2BE475BQGfLbRzXbGysYXK18WgmAyTCRD4DwKg==",
```
Comment (dlockhart) [inline package-lock.json:1331]: You're OK picking up these changes as well?

Comment (Scy-D2L) [reply package-lock.json:1331]: Yep, I doubled checked - it's a minor update that rounds a score to 2 decimals, nothing volatile.

**Review by dlockhart — APPROVED** [review]
Not sure why the one build is stalled or didn't kick off. Might be worth pushing a minor irrelevant change just to see if that wakes it up.

File: package-lock.json (hunk 1 above)
Comment (vix633) [reply package-lock.json:1331]: Yeah these are fine, sorry for the late reply

**Pull request merged**